.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+# Trigger the workflow on push or pull request
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    # Use macOS for emulator hardware acceleration
+    runs-on: macOS-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+ 
+      - name: Gradle Wrapper Validation
+        uses: gradle/wrapper-validation-action@v1   
+        
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1.3.0
+        with:
+          java-version: 1.8
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
```

Comment:
I am node ignorant, any real risk to having this as a dynamic version?

Reply:
Don't have strong preference. I'm not node user. Removed.

Reply:
you didn't have to do that. w/e if we hit issues we can deal with it then.